py-polars/polars/_utils/construction/dataframe.py

```diff
@@ -413,7 +413,7 @@ def sequence_to_pydf(
     infer_schema_length: int | None = N_INFER_DEFAULT,
 ) -> PyDataFrame:
     """Construct a PyDataFrame from a sequence."""
-    if not data:
+    if data is None or len(data) == 0:
```

Comment:
This should be changed to:
```suggestion
    if len(data) == 0:
```